README.md

```diff
@@ -25,6 +25,24 @@ The Cluster API itself is shared across multiple cloud providers allowing for tr
 
 You can find detailed documentation as well as a getting started guide in the [Cluster API Provider for OCI Book][capoci_book].
 
+## 🤗 Community
+
+The CAPOCI provider is developed in the open, and is constantly being improved by our users, contributors, and maintainers.
+
+If you have questions or want to get the latest project news, you can connect with us in the following ways:
+
+- Chat with us on the Kubernetes [Slack](http://slack.k8s.io/) in the [#cluster-api-oci][#cluster-api-oci slack] channel
+
+### Office hours
+
+The community holds office hours on the first Tuesday of every month, with sessions open to all users and
+developers.
+
+Office hours are hosted on a [Zoom video chat][zoomMeeting] on the first Tuesday of every month
+at 06:00 PT / 09:00 ET / 14:00 UK / 18:00 India.
```

Comment:
```suggestion
The maintainers host office hours on the first Tuesday of every month
at 06:00 PT / 09:00 ET / 14:00 CET / 18:00 IST via [Zooom][zoomMeeting].

All interested community members are invited to join us. A recording of each session will be made available afterwards for folks who are unable to attend.
```

I was trying to make it a little more inclusive and a little less repetitive, but I'm not 100% happy with my rewrite.